components/feature/webcompat-reporter/src/main/assets/extensions/webcompat-reporter/background.js

```diff
@@ -41,6 +42,15 @@ async function checkEndpointPref() {
   }
 }
 
+(() => {
+  let port = browser.runtime.connectNative("mozacWebcompatReporter");
+  port.onMessage.addListener(message => {
```

Comment:
It seems you only need this message a single time, right? Should we disconnect the port once the product name is known?

Reply:
Good point, we don't need that port anymore - at least for now. I updated the patch.